packages/lib/src/vis/hooks.ts

```diff
@@ -21,9 +21,9 @@ const useValidDomainForScale = createMemo(getValidDomainForScale);
 export const useAxisDomain = createMemo(getAxisDomain);
 
 export function useDomain(
-  valuesArray: NdArray<number[]> | number[],
+  valuesArray: AnyNumArray,
```

Comment:
We ought to find a better name for this... Or perhaps removing the alias would make things clearer

Reply:
Just to clarify, this is the type definition of `AnyNumArray` (not just `NdArray<number[]> | number[]`):

```ts
export type NumArray = TypedArray | number[];
export type AnyNumArray = NdArray<NumArray> | NumArray;
```

I find the term "any" quite accurate, since it really allows any kind of numeric array: typed array, JS `number` array, ndarray (with typed array of JS `number` array). There are 12 occurrences of it throughout the codebase, which makes it quite useful in my opinion.